test/fixture/report_fixture.go

```diff
@@ -381,6 +382,7 @@ var (
 						kubernetes.PodState:        "running",
 						kubernetes.PodContainerIDs: ServerContainerID,
 						kubernetes.ServiceIDs:      ServiceID,
+						report.HostNodeID:          ServerHostNodeID,
```

Comment:
Oh it didn't; I added it for completeness (all the other nodes have the host ID).